src/Language/Haskell/Liquid/Bare/OfType.hs

```diff
@@ -100,9 +102,10 @@ rtypePredBinds = map uPVar . ty_preds . toRTypeRep
 ofBRType :: (PPrint r, Reftable r)
          => (SourcePos -> RTAlias RTyVar SpecType -> [BRType r] -> r -> BareM (RRType r))
          -> (r -> BareM r)
+         -> (Expr -> BareM Expr)
          -> BRType r
          -> BareM (RRType r)
-ofBRType appRTAlias resolveReft
+ofBRType appRTAlias resolveReft resolveExpr
```

Comment:
It's changing `EVar` to take a `LocSymbol` instead of a `Symbol`, but now I remember why I decided it wouldn't work out. It ends up requiring a large, cascading series of changes to Fixpoint, adding location information requirements where there weren't any before.